app/livechat/client/views/sideNav/livechat.js

```diff
@@ -113,21 +115,28 @@ Template.livechat.events({
 		event.preventDefault();
 		event.stopPropagation();
 
-		modal.open({
-			title: t('Livechat_Take_Confirm'),
-			text: `${ t('Message') }: ${ this.message }`,
-			showCancelButton: true,
-			confirmButtonColor: '#3085d6',
-			cancelButtonColor: '#d33',
-			confirmButtonText: t('Take_it'),
-		}, (isConfirm) => {
-			if (isConfirm) {
-				Meteor.call('livechat:takeInquiry', this._id, (error, result) => {
-					if (!error) {
-						roomTypes.openRouteLink(result.t, result);
-					}
-				});
+		const { rid } = this;
+		Meteor.call('livechat:getFirstRoomMessage', { rid }, (error, result) => {
```

Comment:
Why take the first message instead of the last one?

Reply:
@tassoevan, actually this change keeps the standard implementation of taking livechat inquiries.
Livechat users often send their first question when they start a conversation, so the last one may be out of context.